### PR TITLE
package information links added

### LINKED ISSUE #105: README missing package information
Table in readme.md file is missing references to maven central. (Values there are `TBA`)
Packages are available in individual readmes:

https://github.com/redhat-developer/app-services-sdk-java/blob/main/packages/kafka-management-sdk/README.md#maven-users

We should replace TBA with actual links to packages

### PR DESCRIPTION
**Pull request by himanshu007-creator**
resolves #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ You can view all SDK packages directly in [GitHub](https://github.com/orgs/redha
 
 | Name             | Artifact Id                        | Status | Version           | Package            | Docs               |
 | ---------------- | :--------------------------------- | :----- | ----------------- | ------------------ | ------------------ |
-| kafka            | **kafka-management-sdk**           | beta   | TBA               | TBA                | [doc][kafkagit]    |
-| service registry | **registry-management-sdk**        | alpha  | TBA               | TBA                | [doc][registrygit] |
-| service accounts | **kafka-management-sdk**           | alpha  | TBA               | TBA                | [doc][kafkagit]    |
+| kafka            | **kafka-management-sdk**           | beta   | TBA               | [info](./packages/kafka-management-sdk/README.md)                | [doc][kafkagit]    |
```
Comment (wtrocki) [inline README.md:29]: Do you mind also updating versions?

File: README.md [hunk 2]
```diff
@@ -26,15 +26,15 @@ You can view all SDK packages directly in [GitHub](https://github.com/orgs/redha
...
+| service registry | **registry-management-sdk**        | alpha  | TBA               | [info](./packages/registry-management-sdk/README.md)                | [doc][registrygit] |
+| service accounts | **kafka-management-sdk**           | alpha  | TBA               | [info](./packages/kafka-management-sdk/README.md)                | [doc][kafkagit]    |
```
Comment (wtrocki) [inline README.md:31]: Package should return users to maven central package - we return users to readme that contains that package :D

File: README.md (hunk 1 above)
Comment (himanshu007-creator) [reply README.md:29]: not at all!

File: README.md (hunk 2 above)
Comment (himanshu007-creator) [reply README.md:31]: okay

**Comment by wtrocki** [comment]
Still wrong links. See: https://search.maven.org/search?q=g:com.redhat.cloud

**Comment by himanshu007-creator** [comment]
@wtrocki made the changes. Sorry for so many commits, but thanks to you as well for being patient🙂.
Hope its the final commit for this PR🤞🏼

**Review by wtrocki — APPROVED** [review]

**Pull request merged**